:wq

```diff
@@ -0,0 +1,18 @@
+Const addes to constants.go 
```

Comment:
?

/hold

Reply:
Just deleted this file, sorry accident some how.